docs/source/topologies_openstack.rst

```diff
@@ -245,6 +245,66 @@ Openstack Full Stack
         test_var2: "test_var2 msg is grp2 hello"
         test_var3: "test_var3 msg is grp2 hello"
 
+
+Steps to provision Single Host
+==============================
+
+.. contents:: Topics
+
+.. _Steps to provision Single Host:
+
+Credentials
+```````````
+   * save openstack credentials in ```$WORKSPACE/credentials/clouds.yml``` file using below format
+
+.. code-block:: yaml
+
+   
+   ---
+    clouds: 
+      devstack:
+        auth:
+          username: "admin"
+          password: "Secret123"
+          project_name: "my-tenant"
+          auth_url: "http://192.168.122.33:5000/v2.0"
+          
+Topology
+````````
+* create topology file under ```$WORKSPACE/topologies/openstack_topology``` as show below:
+
+.. code-block:: yaml
+
+   ---
+    topology_name: "osp-test"
+    resource_groups:
+      -
+        resource_group_name: "lp-test"
+        resource_group_type: "openstack"
+        resource_definitions:
+          - name: "test1"
+            type: "os_server"
+            flavor: "m1.small"
+            image: "rhel-6.5_jeos"
+            count: 1
+            keypair: "ci-factory"
+            networks:
+              - "e2e-openstack"
+            fip_pool: "192.168.122.1/24"
+        credentials:
+            filename: "clouds.yml"
+            profile: "devstack"
+
+Provision
+`````````
+
+* provision the above topology
+      
+.. code-block:: bash
+   
+   $ cd $WORKSPACE
+   $ linchpin --creds-path  ./credentials/ up 
+
 .. note::
 
   Source of the above mentioned examples can be found at `Example Topologies <https://github.com/CentOS-PaaS-SIG/linch-pin/tree/master/ex_topo>`_
```

Comment:
Can you remove this line? It's completely incorrect now. I know that you are just updating, but it will mislead a user. Thanks!